fs/fs.go

```diff
@@ -71,6 +71,7 @@ type partition struct {
 	minor      uint
 	fsType     string
 	blockSize  uint
+	uuid       string
```

Comment:
Good catch. Removed the irreverent code. I originally used this to get the partition from uuid by iterating the partition slice, and later changed to use a separate `fsUUIDToDeviceName` map, but didn't clean this up..